runtime/doc/vim_diff.txt

```diff
@@ -159,6 +159,7 @@ Commands:
   |:drop| is always available
   |:Man| is available by default, with many improvements such as completion
   |:sign-define| accepts a `numhl` argument, to highlight the line number
+  |:match| can be invoked before highlight group is defined
```

Comment:
this and the `matchadd ()` note should probably go in the relevant sections of `:help nvim-features-changed` . This section is for new stuff rather than changed stuff.

Though maybe the distinction isn't useful and we could merge the sections (out of scope here).

Reply:
all the three above is is changed stuff tho. This sounds like a question for a separate PR.